scipy/sparse/construct.py

```diff
@@ -756,12 +762,12 @@ def random(m, n, density=0.01, format='coo', dtype=None,
     >>> from scipy.stats import rv_continuous
     >>> class CustomDistribution(rv_continuous):
     ...     def _rvs(self,  size=None, random_state=None):
-    ...         return random_state.randn(*size)
-    >>> X = CustomDistribution(seed=2906)
+    ...         return random_state.random(*size)
```

Comment:
`standard_normal()` would be the replacement for `randn()`.

Reply:
oh yes, thanks!